src/HttpOverStream.NamedPipe/NamedPipeHttpClientBuilder.cs

```diff
@@ -11,7 +11,7 @@ public class NamedPipeHttpClientBuilder
         private string _pipeName;
         private ILoggerHttpOverStream _logger;
         private DelegatingHandler _outerHandler;
-        private TimeSpan? _perRequestTimeout;
+        private TimeSpan? _perRequestTimeout; 
```

Comment:
Probably not intentional?

Reply:
perhaps caused by a `\r\n`. removing ...

Reply:
done